src/models/resultsSerializer.ts

```diff
@@ -128,6 +133,22 @@ export default class ResultsSerializer {
         return saveResultsParams;
     }
 
+    private getConfigForExcel(): Contracts.SaveResultsAsExcelRequestParams {
+        // get save results config from vscode config
+        let config = vscode.workspace.getConfiguration(Constants.extensionConfigSectionName);
+        let saveConfig = config[Constants.configSaveAsCsv];
```

Comment:
Well here's the thing - the option was already set with "csv" in it, but I don't think we should duplicate the "copy with headers" setting. So I'm proposing we leave it alone and use for both.